src/Sylius/Bundle/UiBundle/Resources/config/app/parameters.yaml

```
@@ -0,0 +1,5 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+parameters:
+    sylius_ui.use_webpack: true
```

I would be for defining this parameter in the configuration tree in UiBundle and then change:
```suggestion
sylius_ui:
    use_webpack: true
```

---

@GSadee I wanted to do it this way but I also declare
```yaml
twig:
    globals:
        use_webpack: '%sylius_ui.use_webpack%'
```

When I was setting the parameter in the extension it was complaining about a missing parameter. I've decided if I need to define it anyway to drop the idea about configuration and left only the parameter.

---

It would probably work if you set this parameter in `prepend` method in the extension 🤔

---

I would be 👍 for configuration, as this is indeed what we're doing here - **configuring** how the UI of Sylius should work. We can process this configuration in the bundle extension and make a parameter from it (we're already doing such things for the `sylius_api.enabled` parameter), so it would be consistent with our current implementation and valid from the functional point of view 🖖